docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_label: "Audit Scanner"
+title: "Kubewarden Audit Scanner"
+---
+
+# Kubewarden Audit Scanner
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
```

Comment:
I'm not sure I like making so much point about being useful for resources deployed before the policies, as it supports policies checking for UPDATE or DELETE. I think we should promote the idea that it gives a continuous feedback on the state of the cluster.

Reply:
I understand your point and I will add some information to make that clear. 

Just one note, we do not support `UPDATE` and `DELETE` events. Actually, we are only considering policies that watch for `CREATE` [event](https://github.com/kubewarden/audit-scanner/blob/8d9b30d6fd4c013c0bf146f8d4e22c1a06ed3404/internal/policies/filter.go#L31-L44). Therefore, the audit scanner will always simulate as if the resource is created on that time. Taking a look in the [RFC](https://github.com/kubewarden/rfc/blob/main/rfc/0011-audit-checks.md), we should consider `DELETE` events as well. Therefore, I believe we need to change the filter function and the [code](https://github.com/kubewarden/audit-scanner/blob/8d9b30d6fd4c013c0bf146f8d4e22c1a06ed3404/internal/resources/admission_request.go#L25C5-L25C10) that created the `AdmissionRequest`.